app/views/articles/_index_default.html.erb

```diff
@@ -40,8 +40,10 @@
     <% if document.access_panels.online? %>
       <% document.access_panels.online.links.each do |link| %>
         <li>
-        <% if link.href == 'detail' %>
+        <% if link.href == 'detail' # PDFs %>
```

Comment:
Do we know if these the only PDF links?  I'll 👍 so we can :shipit: to preview, but we may want to look into if non timebombed detail links will also be PDFs and should get this icon.